### PR TITLE
App skeleton

### PR DESCRIPTION
**Pull request by FilipNew1234**
I have just done a Pull request before sending a Merge/Pull request.

Everyone will have to first install react-router-dom

I believe you need to:
1. Approve my merge request (2 approvals needed) 
2. git pull
3. npm install

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,24 @@
+import React from 'react';
 import './App.css';
+import Dashboard from './Pages/Dashboard';
+import { Route, BrowserRouter as Router, Switch } from 'react-router-dom';
+import PrivateRoute from './components/PrivateRoute';
+import Login from './Pages/Login';
 
-function App() {
```
Comment (Edith2019) [inline src/App.js:3]: Are we using functions or classes?

File: src/Pages/Login.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+			<br /> <br /> <br />
+			<h1>Login Page</h1>
+			<br />
+			<label> user name </label>
+			<input type="text"  />
+			<br />
+			<br />
+			<label> password </label>
+			<input type="text" />
+			<br />
```
Comment (Edith2019) [inline src/Pages/Login.js:26]: The pw should not be visible from the oustide but hided with dots or something. 
input type="password"
Screen Shot 2020-11-05 at 12 38 43

File: src/App.js (hunk 1 above)
Comment (FilipNew1234) [reply src/App.js:3]: we agreed on using only functional components

Comment (FilipNew1234) [reply src/App.js:3]: juts good to understand Class components, but they are getting outdated, as Hooks are the future :)

File: src/Pages/Login.js (hunk 2 above)
Comment (FilipNew1234) [reply src/Pages/Login.js:26]: ah yeah, I just created the easiest thing I could, just so the router works. Now we can start working on this

**Review by Edith2019 — APPROVED** [review]

**Review by HelenHMaher — APPROVED** [review]
looks like a good starting point

**Comment by HelenHMaher** [comment]
I think the most painful part is going to be this first bit where we're trying to agree on how we're doing things and getting a basic structure in place.  This looks like a good routing strategy to work from.

**Comment by FilipNew1234** [comment]
> I think the most painful part is going to be this first bit where we're trying to agree on how we're doing things and getting a basic structure in place. This looks like a good routing strategy to work from.

yeah, anyone feel free to remove the Dashboard and start making amendments

**Pull request merged**